manifests/3.0.0/opensearch-3.0.0.yml

```diff
@@ -238,3 +238,9 @@ components:
       - anomaly-detection
       - sql
       - ml-commons
+  - name: user-behavior-insights
```

Comment:
What is the depends_on entry for this plugin?

Reply:
Are the `depends_on` other OpenSearch plugins?